.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+
+* Paul Bahr from Peachtree Sound will send the edited podcast and show notes \(email\) **After podcast is edited**
+  * [ ] **Action:** Get edited podcast and show notes from Paul
+  * [ ] **Action:** Review the Episode Planning Document for the guest's profile pic, bio, and social media links.
+  * [ ] **Action:** Create the Cover Art by downloading a sample [cover art from Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AADj2oAVfg2brvd8DulJPxEla/Art?dl=0&subfolder_nav_tracking=1) and updating the episode number
+  * [ ] **Action:** Upload the shownotes to google drive and use Convert to Markdown to get markdown format
+  * [ ] **Action:** In [Fireside](https://chaosscast.fireside.fm/admin/), create missing panelists and guests
+    * Slug is `firstname-lastname` \(e.g., `georg-link`\)
+    * Status: public
+    * The other information should be in the Episode Planning Document
+  * [ ] **Action:** In [Fireside](https://chaosscast.fireside.fm/admin/), create a new episode
+    * Permalink: \# \(only the number of episode\)
+    * Visibility: Public
+    * Title: Title from the Episode Planning Document
+    * Title Format: Episode 123: My Great Title \(default\)
+    * Content: Clean \(unless we used swear words\)
+    * Summary: First paragraph from the Show Notes are a good starting point
+    * Description: Insert the show notes in Markdown format.
+      * Hint: convert show notes to Google Doc and export as Markdown
+    * Keywords: \(use sparingly to avoid keyword cannibilization\)
+    * Tags: \(empty\) -- we can use tags later for having sub-podcasts
+    * Hosts and Guests: activate everyone who was on the show. Regular panelists are all hosts, even when they are guest on an episode.
+    * Social Media Destinations: \(ignore\)
+    * Apple Podcasts Settings: \(ignore\)
+    * MP3 File: upload [MP3 file provided by CPN through Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AAC8aNnZYbKy6K6gsdg5pfTGa/Episodes?dl=0&subfolder_nav_tracking=1)
+    * Cover Art: upload [cover art from Dropbox](https://www.dropbox.com/sh/l7iv7qewxk5ud8o/AADj2oAVfg2brvd8DulJPxEla/Art?dl=0&subfolder_nav_tracking=1) with the correct episode number
+    * Header Image: \(ignore, will use default\)
+    * Transcript: \(ignore\)
+  * PUBLISH
+    * [ ] **Action:** Update the template [CHAOSScast Calendar](https://calendar.google.com/calendar?cid=NTEzcjQ3YmI3NTZ0MDdscDV0YXQ1bnN1ajBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) event \(🎧 \(TBD\) Release CHAOSScast episode\) for the release date \(i.e., remove `(TBD)` and add epsiode number and title\)
+    * [ ] **Action:** Post about it on Discourse and Slack \([template](https://docs.google.com/document/d/1TyZAX7xfYmx3gKSFT2i2K5ERIA1ibGAQVfXMnGmFYyg/edit)\)
+    * [ ] **Action:** Include in upcoming [CHAOSS newsletter](https://docs.google.com/document/d/1N2dbrqHYSxIUF1vXZ1-854aaoGIf86uey0XRhxhgG4o/edit)
+    * [ ] **Action:** Check ITTT automated \(RSS feed publishing\):
+      * Schedule a tweet through Tweetdeck
```

Comment:
You can also remove this step; we don't tweet anymore. And we no longer use Tweetdeck to my knowledge.